sources/install/package_wordlists.sh

```diff
@@ -106,6 +105,52 @@ function install_genusernames() {
+    wget https://github.com/NSAKEY/nsa-rules/raw/refs/heads/master/_NSAKEY.v2.dive.rule -P /opt/rules/
+    wget https://raw.githubusercontent.com/praetorian-inc/Hob0Rules/refs/heads/master/d3adhob0.rule -P /opt/rules/
+    wget https://raw.githubusercontent.com/praetorian-inc/Hob0Rules/refs/heads/master/hob064.rule -P /opt/rules/
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/hashesorg.v6/pantagrule.hashorg.v6.hybrid.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.hashorg.v6.hybrid.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/hashesorg.v6/pantagrule.hashorg.v6.one.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.hashorg.v6.one.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/hashesorg.v6/pantagrule.hashorg.v6.popular.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.hashorg.v6.popular.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/hashesorg.v6/pantagrule.hashorg.v6.random.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.hashorg.v6.random.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/hashesorg.v6/pantagrule.hashorg.v6.raw1m.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.hashorg.v6.raw1m.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/private.hashorg.royce/pantagrule.popular.royce.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.popular.royce.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/private.hashorg.royce/pantagrule.hybrid.royce.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.hybrid.royce.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/private.hashorg.royce/pantagrule.one.royce.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.one.royce.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/private.hashorg.royce/pantagrule.random.royce.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.random.royce.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/private.v5/pantagrule.private.v5.hybrid.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.private.v5.hybrid.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/private.v5/pantagrule.private.v5.one.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.private.v5.one.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/private.v5/pantagrule.private.v5.popular.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.private.v5.popular.rule.gz
+    wget https://github.com/rarecoil/pantagrule/raw/refs/heads/master/rules/private.v5/pantagrule.private.v5.random.rule.gz -P /opt/rules/
+    gunzip /opt/rules/pantagrule.private.v5.random.rule.gz
+}
```

Comment:
Can you add a add-test-command and add-to-list ?

Reply:
only for this 2 gunzip ?